test/fixed/6.Operators.php

```diff
@@ -83,11 +83,12 @@ public function testUseStrictComparisonOperators(): void
     {
         // Loose comparison operators SHOULD NOT be used, use strict comparison
         // operators instead.
+        // Fix that manually.
 
-        $foo === 123;
-        123 === $foo;
-        true !== 0.0;
-        false !== true;
+        $foo == 123;
+        123 == $foo;
+        true != 0.0;
+        false <> true;
```

Comment:
Please don't remove the old test cases, just add these new ones. This way we are sure that we can use both. We don't want to change strict comparison to loose comparison where it is used.

Reply:
Here test checks fixer. phpcs test based on warning numbers in report of cs-check